.ci/bump-version.sh

```diff
@@ -1,9 +1,68 @@
 #!/bin/bash
 set -e
 
-VERSION=$1
-echo VERSION="$VERSION"
+if [[ -z $1 ]]; then
+  echo "version is not set"
+  echo "Usage: $BASH_SOURCE <version>"
+  exit 1
+fi
+
+NEW_VERSION=$1
+echo NEW_VERSION="$NEW_VERSION"
+
+if ! [[ "$NEW_VERSION" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+  echo "[ERROR] New version($NEW_VERSION) is not in the format of <major>.<minor>.<patch>"
+  exit 1;
+fi
+
+CURRENT_VERSION=$(xmlstarlet sel -N pom=http://maven.apache.org/POM/4.0.0 \
+                             -t -m pom:project -v pom:version pom.xml | sed "s/-SNAPSHOT//")
```

Comment:
Why can't we replace this with call to `getCheckstylePomVersion` from util.sh?

Reply:
No issues in doing it. It just takes 1-2 seconds longer.

I saw that we have other places where this `xmlstarlet` call can be replaced with a call to new function `getCheckstylePomVersionWithoutSnapshot`. Shall I open an issue for this?

Done. Replaced.

Reply:
Please do.